api/systembackup.go

```diff
@@ -23,9 +26,15 @@ func (s *Server) SystemBackupCreate(w http.ResponseWriter, req *http.Request) er
 		return err
 	}
 
+	backupTarget, err := s.m.GetBackupTarget(types.DefaultBackupTargetName)
+	if err != nil {
+		return errors.Wrapf(err, "failed to get default backup target")
+	}
+
 	obj := &longhorn.SystemBackup{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: input.Name,
+			Name:            input.Name,
+			OwnerReferences: datastore.GetOwnerReferencesForBackupTarget(backupTarget),
```

Comment:
Do we need to handle the existing system backups while upgrading Longhorn?

Reply:
Added when upgrading.